### PR TITLE
Lab4

### PR DESCRIPTION
**Pull request by 0xD1M0N**

### CODE REVIEW

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
remove all bin files

**Comment by 0xD1M0N** [comment]
> remove all bin files

Done

File: Lab4/SW (Client)/minimax/__init__.py
Comment (andrii-s-spitzer-lpnu-ua) [inline Lab4/SW (Client)/minimax/__init__.py:1]: why do you need an empty file?

Comment (0xD1M0N) [reply Lab4/SW (Client)/minimax/__init__.py:1]: To mark a directory as a Python package

**Review by andrii-s-spitzer-lpnu-ua — APPROVED** [review]

**Pull request merged**